README.md

```diff
@@ -1,39 +1,65 @@
-# Temporal  
 [![Build status](https://badge.buildkite.com/fc0e676d7bee1a159916af52ebdb541708d4b9f88b8a980f6b.svg)](https://buildkite.com/temporal/temporal-server)
 [![Coverage Status](https://coveralls.io/repos/github/temporalio/temporal/badge.svg?branch=master)](https://coveralls.io/github/temporalio/temporal?branch=master)
 [![Discourse](https://img.shields.io/static/v1?label=Discourse&message=Get%20Help&color=informational)](https://community.temporal.io)
 
-Visit [docs.temporal.io](https://docs.temporal.io) to learn about Temporal.
+# Temporal  
+
+Temporal is a microservice orchestration platform which enables developers to build scalable applications without sacrificing productivity or reliability.
+Temporal server executes units of application logic, Workflows, in a resilient manner that automatically handles intermittent failures, and retries failed operations.
 
-This repo contains the source code of the Temporal server. To implement workflows, activities and worker use [Go SDK](https://github.com/temporalio/sdk-go) or [Java SDK](https://github.com/temporalio/sdk-java).
+The programming model of Temporal exposed via the SDKs (currently for Java and Go) offers developers the simple abstractions of Workflows and Activities that can virtually "resume" execution after a failure from the same line of code.
```

Comment:
> "resume" execution after a failure from the same line of code

for workflow, mostly yes
for activity, not really, activity can have the progress of last heartbeat, but not really able to resume from the same line of code

Reply:
Updated.